clear-site-data/support/controlled-endpoint.py

```diff
@@ -1,3 +1,3 @@
 def main(request, response):
-    return ([("Content-Type", "text/html")],
-            "FROM_NETWORK")
\ No newline at end of file
+    return ([(b"Content-Type", b"text/html")],
+            u"FROM_NETWORK")
```

Comment:
Could you add a newline here if it's not too much trouble?

Reply:
Added.